.circleci/config.yml

```diff
@@ -37,6 +37,14 @@ executors:
       - image: *image
       - image: cimg/mysql:5.7
 
+  sqlite:
+    working_directory: *workdir
+    environment:
+      <<: *environment
+      DB: sqlite
+    docker:
+      - image: *image
+
```

Comment:
Do you think we should add this to the https://github.com/solidusio/circleci-orbs-extensions repo as well so extensions also can opt-in to running specs against SQLite on CI?

Reply:
Added in https://github.com/solidusio/circleci-orbs-extensions/pull/48, will be released in v0.4.0